_posts/2016-11-09-foreman-community-newsletter-january-2016.md

```diff
@@ -0,0 +1,71 @@
+  - [hammer-cli-foreman](https://github.com/theforeman/hammer-cli-foreman) updated to 0.5.1 on 14th of December
+
+### New media and blogs
+
+- The latest [community demo](https://www.youtube.com/watch?v=gvOFbmZO1yM&list=PLLTIBSsvp9qQwNxhQVtaqNNkMkvsHldGA) took place on December 10th. Highlights:
+  - [Bare metal image-based provisioning (lzap)](https://youtu.be/gvOFbmZO1yM?t=0)
+  - [Remote execution: CLI (stbenjam)](https://youtu.be/gvOFbmZO1yM?t=575)
+  - [Remote execution: Recurrence (aruzicka)](https://youtu.be/gvOFbmZO1yM?t=840)
+  - [Remote execution: Effective user (inecas)](https://youtu.be/gvOFbmZO1yM?t=1220)
+  - [Parameter UI improvements (tbrisker)](https://youtu.be/gvOFbmZO1yM?t=1675)
+  - [Hammer: Setting parameters for Taxonomies (tstrachota)](https://youtu.be/gvOFbmZO1yM?t=2062)
+  - [Hammer: Setting CLI defaults (aglodboi)](https://youtu.be/gvOFbmZO1yM?t=2256)
+  - [Community updates (gwmngilfen)](https://youtu.be/gvOFbmZO1yM?t=2785)
+
+- We held a deep dive about [Taxonomies](https://www.youtube.com/watch?v=D4cON77hmnI) with Daniel Lobato Garcia, covering used Models, how they interact with Roles and Filters and how they interact with other plugins. Thanks Daniel!
+
+### Upcoming events
+- There is another [Deep Dive](https://www.youtube.com/watch?v=0BSnlUkCC7I) planned for Tuesday January 19th, where Dmitri Dolguikh will be talking about the Smart Proxy, aiming to provide an overview of proxy plugins and plugin providers, validators, events and event reporting.
+- [FOSDEM](https://fosdem.org/2016/) takes place on 30th and 31st of January.
+- [Config Management Camp](http://cfgmgmtcamp.eu/) takes place on 1st and 2nd of February and Foreman will have dedicated track there. List of Foreman related talks from all tracks includes:
+  - Case study: Dynamic management of heterogeneous test environments using Foreman by Shimon Stein 
+  - Hosts' Lifecycle with Ansbile and Foreman by Daniel Lobato
+  - Community Update by Greg Sutcliffe
+  - Intro to parameters in Foreman by Tomer Brisker
+  - Demystifying the Foreman by Julien Pivotto
+  - Foreman and remote execution by Marek Hulan
+  - Security and Compliance reports in foreman by Shlomi Zadok
+  - PXEless discovery by Stephen Benjamin
+  - There's a plugin for that by Greg Sutcliffe
+  - Extending operating systems support of Foreman by Michael Moll
+  - Foreman and PowerDNS by Ewoud Kohl van Wijngaarden
+  - Infrastructure as code, foreman_provision by Nils Domrose
+  - SaltStack integration with Foreman by Stephen Benjamin
+
+- We will be holding the [annual Foreman dinner](https://groups.google.com/forum/#!topic/foreman-users/62yYbdDlojU) on January 30th in Brussles. Please sign up via [G+](https://plus.google.com/events/co1nia0c7g6hk9b4msf98517s1o) or by mail to [Greg](mailto:greg.sutcliffe@gmail.com) if you wish to join us, so that we know how many seats to reserve at the restaurant.
```

Comment:
Bad news, apparently we had this one in the last newsletter too. I just copied this part since it still applies. Will fix nevertheless